quantinuum_sphinx/_templates/page.html

```diff
@@ -189,98 +189,19 @@
           {% block content %}{{ body }}{% endblock %}
         </article>
       </div>
-      <footer>
-        {% block footer %}
-        <div class="related-pages">
-          {% if next -%}
-            <a class="next-page" href="{{ next.link }}">
-              <div class="page-info">
-                <div class="context">
-                  <span>{{ _("Next") }}</span>
-                </div>
-                <div class="title">{{ next.title }}</div>
-              </div>
-              <svg class="furo-related-icon"><use href="#svg-arrow-right"></use></svg>
-            </a>
-          {%- endif %}
-          {% if prev -%}
-            <a class="prev-page" href="{{ prev.link }}">
-              <svg class="furo-related-icon"><use href="#svg-arrow-right"></use></svg>
-              <div class="page-info">
-                <div class="context">
-                  <span>{{ _("Previous") }}</span>
-                </div>
-                {% if prev.link == pathto(master_doc) %}
-                <div class="title">{{ _("Home") }}</div>
-                {% else %}
-                <div class="title">{{ prev.title }}</div>
-                {% endif %}
-              </div>
-            </a>
-          {%- endif %}
-        </div>
-        <div class="bottom-of-page">
+      <footer style="bottom:0;z-index:1024;width:100%;font-size: var(--font-size--small);">
+        <div style="display:flex;align-items:center;justify-content:space-between;color:var(--color-foreground-secondary);">
           <div class="left-details">
-            <div class="copyright">
-              Copyright &#169; 2024 Quantinuum Ltd. All rights reserved.
-            </div>
-            <div class="terms" style="display:flex; gap:0.5rem; align-items:center; margin: 0.25rem 0rem;">
-              <a href="https://www.quantinuum.com/privacy-statement" target="_blank">Privacy Statement</a>
-              <div role="separator">/</div>
-              <a href="https://www.quantinuum.com/cookie-notice" target="_blank">Cookie Notice</a>
-              <div role="separator">/</div>
-              <a href="https://www.quantinuum.com/terms-conditions" target="_blank">Terms and Conditions</a>
-            </div>
-            {% trans %}Made with {% endtrans -%}
-            {%- if show_sphinx -%}
-            {% trans %}<a href="https://www.sphinx-doc.org/">Sphinx</a> and {% endtrans -%}
-            <a class="muted-link" href="https://pradyunsg.me">@pradyunsg</a>'s
-            {% endif -%}
-            {% trans %}
-            <a href="https://github.com/pradyunsg/furo">Furo</a>
-            {% endtrans %}
-           
-            {%- if last_updated -%}
-            <div class="last-updated">
-              {% trans last_updated=last_updated|e -%}
-                Last updated on {{ last_updated }}
-              {%- endtrans -%}
```

Comment:
I would leave all the default content as is, apart from things we are making custom. It makes it clearer what we have modified vs what was original. For example, the furo attribution should be left untouched imo.